src/index.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-unused-vars */
+import _ from 'lodash';
+import './style.css';
+
+const list = document.getElementById('list');
+
+const doList = [
+  {
+    id: 0,
+    description: 'Go to the market',
+    completed: false,
+  },
+  {
+    id: 1,
+    description: 'Clean your home',
+    completed: false,
+  },
+  {
+    id: 2,
+    description: 'Programming',
+    completed: false,
+  },
+];
```

Comment:
- [ ] _On page load, the task array should be rendered dynamically to the DOM. The tasks should appear in order of the index values for each task._ You have implemented everything just as asked only the manually added tasks should appear in the browser.
![Screenshot from 2022-01-11 17-04-33](https://user-images.githubusercontent.com/62705947/148978092-d7eede40-2090-4b17-9267-22e546033162.png)